pnparser/helpers.py

```diff
@@ -0,0 +1,42 @@
+import re as Regex
+from bs4 import Tag
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Iterator
+    from re import Match
+
+
+EXCEPTIONS: 'list[str]' = ["and", "or", "the", "a", "of", "in", "ARAM", "QoL"]
+
+
+class Helper:
```

Comment:
StringHelper?

Reply:
if you rename this, use pycharm to do it - right-click and you can rename the class, I can't remember the default hotkey for it, I rebound to Ctrl+R ctrl+R (double key-press of R basically) (apparently that's the default keybind in Resharper, a friend suggested it to me when I said I didn't like the default) Anyway, Pycharm will auto fix every import and usage in the entire project.

Reply:
StringHelper seems better. It is done.